VimeoNetworking/Sources/Models/VIMVideo.m

```diff
@@ -566,4 +572,9 @@ - (BOOL)canDownloadFromDesktop
     return false;
 }
 
+- (BOOL)hasFileTransferPage
```

Comment:
I wonder if we should change the name of this to be more specific to the logic below. Since a video can have a page, but we return false if it can't be downloaded. Something like `allowsFileTransfer` or `supportsFileTransfer`